---
title: Notes on miRNA-seq analysis
date: 2016-03-03T09:27:54+00:00
layout: post
categories:
  - gsta mirna
---
Challenges:

  * Dataset contains more that just mature miRNAs. There may be segments of precursors and other longer mRNAs, as well as other short RNAs.
  * Many of the same problems as a GBS dataset with no reference genome.
      * Aligning to just the genome won't capture miRNAs where the precursors had introns (how common is this?)
      * Aligning to just known miRNAs won't capture novel miRNAs
      * Assembling to a pseudo-reference generated by clustering involves problems with determining whether similar (but not exactly matching) stacks represent SNPs in a single miRNA locus or multiple similar loci.
  * Short reads are going to have higher rates of non-specific alignment regardless of the reference used.
  * Depth of coverage will likely be very high, but noise level will also be high.

The CAP-miRSeq process flow [ doi:10.1186/1471-2164-15-423 ]( http://doi.org/10.1186/1471-2164-15-423 ) aligns against both the reference genome and known miRNAs. Would it be worth also aligning against a clustered pseudogenome?

![ CAP miRSeq flowchart ]( {{site.image_path}}wp-content/uploads/2016/03/CAP-miRSeq_flowchart.png )

  * runs on Linux using many tools I'm already familiar with
  * parallel processing
      * total time is the time to run one sample + time to merge results at the end
      * 4-5 hours typical
      * 5-10GB memory (no problem on the workstation, possibly even on the USU cluster)
  * Captures info on non-miRNAs too
  * Looks for both SNPs in miRNAs and expression differences
