---
title:  Sphagnum reading
date:  2018-11-08
layout: post
categories:
  - sphagnum

---

# Readings from Jon Shaw

[Review and timeline][1] of Amazonian geologic and biotic diversification history.

[Norm Wickett et al's massive phylogenomic analysis][2] of algae and landplants. Mosses and liverworts are probably monophyletic but it isn't clear if they are:
  * sister to vascular plants with hornworts sister to all that -  h,((m,l),v), or
  * sister to hornworts and vascular plants - (m,l),(h,v), or
  * monophyletic with hornworts and all that is sister to vascular plants - ((m,l),h),v

The previously accepted order of bryophytes as successive sister groups to landplants wasn't well supported in this study - l,(m,(h,v))

[_Sphagnum palustre_ morphological diversity in the Azores][3] using microsats. There is more morphological diversity in the Azores than in mainland European and American populations (3 color morphs), but the diversity doesn't seem to have a genetic basis and individuals of different morphs on the same island are more genetically similar than individuals of the same morph on different islands, suggesting high levels of plasticity in island populations. It isn't clear if this is because there is higher environmental variability on islands compared to mainland or that colonizers of variable and stressful islands are selected for high plasticity.

[Sphagnum organellar phylogeny][4] using all mitochondrial and chloroplast genes. Five subgenera: (Rigida, (Subsegunda, Cuspidata), (Sphagnum, Acutifolia)). Several allopolyploid hybrids including some inter-subgeneric hybrids and one apparent haploid hybrid.

Cuspidata and Subsecunda tend to be species of hollows, whereas Sphagnum and Acutifolia tend to form hummocks. Rigida is less common in peatlands and the outgroups (Eosphagnum and Flatbergia don't occur in peatlands.

[Timing of bryophyte diversification][5] using multiple genes from all 3 genomic partitions and calibrating with fossils. Overall, diversification rates in mosses are lower than in ferns or angiosperms, but recently diverged groups have comparable rates. Since current rates are similar to other plant groups it is likely the lower overall diversification rates are being driven by past extinctions. The data seem to support the "In the shadow of the angiosperms" hypothesis, with lots of diversification in mosses since the angiosperms appeared.

[Sphagnome project summary][6].

Genomic tools being developed:
  * Two reference genomes: _S. fallax_ and _S. megellanicum_.  
    * Resequencing for 15 additional species.  
    * Shallow resequencing of 200 _S. fallax_ haploid siblings.

Major questions to be investigated:
  * "What is the biological basis of unique Sphagnum traits or combinations of traits, and how do these trait combinations extend beyond the organism?"
  * "Did adaptation to spatially or temporally varying climate variation spark Sphagnum species radiations?""
  * "What are the factors that limit or facilitate local-scale adaptive evolution?""
  * "What is the role of Sphagnum and its interacting microbiome in ecosystem carbon and nitrogen cycling?"
  * "How do we model Sphagnum genotype-by-environment interactions?"

[Factors controlling mating patterns in Sphagnum][7].

[Worldwide patterns of population diversity in Sphagnum][8]. Some species show no genetic structure suggesting frequent long-distance spore dispersal. In others, mountains seem to be a bigger dispersal barrier than oceans, with NE North American/European plants more genetically distant than Pacific Northwest/Asian plants. There also appears to be a N-S separation between Alaskan and Pacific NW plants in some spp.

[Testing for phylogenetic differences in Sphagnum species' niche preferences][9]. There is a phylogenetic component to differences in height above water table (the hummock-hollow gradient) and possibly pH but not for other ionic gradients.

[Looking at margin and expanse morphs of S. magellanicum][10]. The colors change with environment (phenotypic plasticity) but the other morphological characters have a genetic basis and the two morphs form separate genetic clusters.

[Book chapter on Sphagnum genome project][11].

Comparative size for reference:
  * Arabidopsis 135 Mbp
  * Sphagnum 450 Mbp
  * Fern (small- Azolla) 750 Mbp
  * Human 3.2 Gbp
  * Fern (average) 12 Gbp
  * Fern (extreme) 148Gbp

__Unpublished moss backbone manuscript__ (Exome capture and 1KP data phylogeny using genes from all 3 genomic partitions). Sphagnopsida diverges early... probably right after Takakiopsida (the earliest diverging) but some data also supports either Sphagnopsida diverging first or a monophyletic Sphagnopsida + Takakiopsida. 400 million year old divergences are hard to tease out, even with genomic data.

__Unpublished Sphagnum biogepgraphy manuscript__ (4 plastid loci).  

[1]: https://doi.org/10.1126/science.1194585
[2]: https://doi.org/10.1073/pnas.1323926111
[3]: https://doi.org/10.1639/0007-2745-117.2.095
[4]: https://doi.org/10.1093/aob/mcw086
[5]: https://doi.org/10.1038/ncomms6134
[6]: https://doi.org/10.1111/nph.14860
[7]: https://doi.org/10.1111/bij.12497
[8]: https://doi.org/10.1111/jbi.12716
[9]: https://doi.org/10.1111/evo.12547
[10]: https://doi.org/10.3732/ajb.1700163
[11]: https://doi.org/10.1016/bs.abr.2016.01.003
