---
title: Notes while reading the Turkey GST Grant and related topics
date: 2014-11-13T09:06:11+00:00
layout: post
categories:
  - gsta mirna
  - gsta methylation
---
Need more background on:

  * miRNA silencing
  * methylation, CpG islands
  * regulation of gene expression in general, and of GSTs in particular
  * epigenetics in general
  * qRT-PCR
  * immunoinhibition experiments in Objective 1 – they use as antiserum that “blocks” a specific one of the GSTs, blocking each of them one at a time to see which one makes the biggest difference?
  * Nrf2/Keap1 regulation of GSTs in mammals and chickens
  * methylated DNA immunoprecipitation with methyl-cytosine antibody followed by a promotor tilling microarray (MeDIP-Chip, Guerrero-Bosagna et al 2010, Weng et al 2009)
  * Mass spectrometry-bisulfite analysis for mapping CpG DNA methylation and differential DNA methylation regions
  * RNA-seq
  * VELVET, DAVID, BOWTIE, TOPHAT
  * HPLC
  * Elisa test

Reading notes:

  * Aflatoxins produced by Aspergillus fungi
  * Used in the study:
      * wild turkeys (Rio Grande and Eastern subspecies)
      * Heritage turkeys (Royal Palm)
      * Modern domestic chickens (Nicholas line)
  * Turkey genome (Meleagris gallopavo),
      * 33 chromosomes,
      * 1Gb
      * ~14000 genes
      * Genbank: GCA_000146605.1
      * GSTA3 is on chromosome 2
      * P450′s are on chromosome 12 & 16
  * Available genomic tools:
      * domestic turkey genome
      * chicken genome
      * chicken reference mRNA sequence gene set
      * eastern wild turkey genome (Reed)? At least the BAC with the GST coding sequences
  * miRNA silencing:
      * post-transcriptional – does this mean that if this is how GSTs are being repressed in domestic turkeys, that we would still expect to find high levels of GST RNAs using methods like RNA-seq or RT-PCR?
  * Methylation & gene expression:
      * blocks transcription by interfering with binding of transcription factors or leading to modification of chromatin structure
      * bisulfite sequencing or methylation-specific PCR detect methylation by replacing unmethylated CpGs to UpGs.
      * HELP assay compares fragment lengths after digesting with 2 RE’s, one of which only digests unmethylated sites
      * Reduced representation bisulfate sequencing enriches for parts of the genome that are high in CpG content
      * Methylated DNA immunoprecipitation (MeDIP) –  shear the DNA, use immunoprecipitation to enrich for fragments containing methylated CpGs. Then sequence and align to the genome to find where enriched fragments are located, or apply both the enriched and unenriched DNA to an array chip to look for binding differences
      * COBRA – combined bisulfite restriction analysis
