---
title: Methylation questions for Clay Isom
date: 2014-12-04T14:45:48+00:00
layout: post
categories:
  - gsta methylation
---
  * The paper mentions that the 14 regions they sequenced were identified in a small pilot study that identified regions with potentially different methylation between their samples. What method did that use? Just a shotgun sequencing approach?
  * Why are C's acceptable in the reverse primers? Isn't a CpG on one strand also a CpG on the reverse complement and isn't there the possibility it could be methylated on that strand as well? Or are we assuming that the reverse primer may not be active in the first round of PCR (the first round just amplifies one strand)? (Does that even make sense?)
  * The different samples and amplicons were identified using the standard Illumina indexing? It says they made 2 libraries (one for the invivo embryos and one for the parthenogenic. So were they not able to differentiate sequences from samples in each library or were the samples individually indexed? If they were individually indexed, why do it as two libraries if they were to be run on a single lane anyway?
