---
title: Notes from Next Generation Pteridology Conference in Washington, DC
date: 2015-06-09T10:38:48+00:00
layout: post
categories:
  - conference-travel
  - t. intricatum gbs
---
June 1-6 2015 at the Smithsonian Natural History Museum

  * "Your hypothesis is shit" - Henriette Schluepmann (Utrecht University)
      * My hypothesis is backward so I end up with a negative finding (I hypothesized Isolation By Distance and failed to find it). I need multiple hypotheses so I can have a positive result
      * How do I know my lack of IBD isn't because I have bad sequence (no data or just too much noise)?
      * How do I know my homozygous loci aren't just heterozygotes with missing alleles (Carl Rothfels raised this question)?
  * Ploidy - Don found fixed heterozygosity with allozymes - more evidence for diploid gametophytes
  * Need to extract rbcL and rps4 sequences from each sample and confirm identity. Compare to known T. boschianum and D. petersii to make sure I can differentiate species this way.
  * Could I get at ploidy or hybrid parentage using GapCp or other nuclear markers?
  * Is there evidence of IBD in T. boschianum or D. petersii?
  * How different are pairs of most similar samples to each other? Do I have things that look like they may be the same multilocus genotypes but with accumulated mutations (i.e., lineages sharing an origin)?
  * Can I do a plot showing the number of steps between each of my genotypes to help identify with are most likely descended from a single spore or parent sporophyte population? - David Wickell did something like this with his asexual sporophyte populations.
  * Make better use of PCA and look into PCOMC (PCOCMC?) - Emailed David Wickell to find out what the PCA-like method he used was, since I can't find any info on PCOCMC.
  * These are asexual lineages accumulating mutations. If there is no local dispersal at all what would I expect to find?
  * Can I do something similar to a mantel test for IBD but instead shuffling genotypes to show that my data differ from random noise?
  * Can we use the current SNP data along with similar data from the Asian species to look for evidence of relaxed selection? Some of these loci are in genes so I could look at dN/dS or maybe just compare rates of sequence variation. - suggested by George Yatskievych (Missouri Botanical Garden)
  * I need to pull together my multiple rps4 and rbcL loci (that are all most similar to Crepidomanes intricatum/schmidtianum in Genbank and figure out why they didn't join. Did I not cluster at high enough stringency?
  * How much genetic variation would be expected to accumulate in a slow-reproducing asexual in 15000 years? This could be another null hypothesis to test.
  * Some of my samples may represent multiple clones. How can I identify this?
  * When clustering, I had the unique reads in order of decreasing size to make sure my seeds were the longest possible sequences, but I also need to sort them by how many copies there are so that my seeds are more likely to be "true" reads rather than reads with errors. This will minimize the distance in my clusters and hopefully give better clusters.
  * Look into Pagel's Lambda as a measure of phylogenetic signal. Example: Does state of residence correlate with the neighbor-joining tree.
  * I may not be able to get enough high-quality DNA for GBS but I can get rbcL and rps4 from herbarium samples. So I need to make a request to do some destructive sampling of Hymenophyllum wrightii sporophytes from the collections at University of British Columbia. Melanie Link-Perez does sampling from even 80 year old herbarium samples. She recommends using a method that includesÂ TBT-PAR for dealing with recalcitrant DNA. <a href="http://doi.org/ http://dx.doi.org/10.3732/apps.1200236">doi: http://dx.doi.org/10.3732/apps.1200236</a>
  * Joel Nitta is interested in seeing my DNA extraction protocol for filamentous gametophytes and pictures of my grinding rig.
  * One of the talks had an interesting way of presenting anova interactions that could be useful on our turkey GSTA data. There are only so many possible patterns of expression change between two groups with 2 treatments so they counted up how many genes had each pattern.
