---
title: GBS ploidy meeting
date: 2015-10-26T23:59:48+00:00
layout: post
categories:
  - t. intricatum gbs
---
Questions:

  * In Karen Mock and Zach Gompert's analysis of the 1:1, 2:1, and 1:2 read proportions, how did they decide what read proportions fit each category and how many reads were required for a locus in an individual to include it (loci with low numbers of reads might introduce a lot of random noise)?
  * If the true ratio is 1:1, only for half of the loci (the ones with even number of reads) can do we even have the possibility of drawing a 1:1 sample. How close we can get to that 1:1 ratio depends on the number of reads and at low read counts it is not very close at all. Same problem for a true 2:1 or 1:2 ration but now only 1 in 3 loci has the possibility of drawing the true ratio. But would this sampling error affect 2:1 and 1:2 loci differently?
  * Sequencing/PCR error would affect 2:1 and 1:2 loci differently:
      * A true diploid that is actually homozygous but has sequencing/pcr error in some reads would be more likely to look like a 2:1 heterozygote than a 1:1 or a 1:2 (assuming the allele in the reference genome is in the reference genome because it was the most common allele or is really the only allele and everything else is an error). But this is the opposite pattern from what we see since there in an excess of diploids that appear to be 1:2's.
  * Given that the range of proportions for diploids and triploids are so similar (even if the averages of all individuals are different) how well can this be applied to assigning an unknown individual to a ploidy level?
  * The ranges of these distributions are pretty big relative to the differences between the means. Are the differences in means really statistically significant?
  * Where did the reference genome come from? If there are multiple alleles in these organisms what decided which allele was used in the reference? Is it biased toward the most common allele in the overall dataset or in some randomly chosen individual?
