---
title: GBS meeting
date: 2013-02-04T12:04:54+00:00
layout: post
categories:
  - t. intricatum gbs
---
Attending: Hardeep Rai, Martin Schilling, Karen Mock, Paul Wolf

  * We are trying to understand how the 2-enzyme with a Y-adapter process results in further genome reduction than just using a single enzyme.  If your second enzyme cuts more often than the single enzyme it seems like you would still get more fragments than if you used one enzyme because the more common enzyme is likely to fall between each of the less common enzyme sites.
  * Adding a nucleotide or two to our primers to reduce the genome futher (AFLP style) won't work because the high-fidelity taq blows right over the mismatch.  But Karen suggested that perhaps we could add 3 or 4 nucleotides which would reduce too much, but then do it will several different sets of 3 or 4 nucleotides to bring the reduction back up to a reasonable level?
